### DISCUSSION TITLE
Thanks for all your work, and question on Digirig

### DISCUSSION THREAD

**Discussion by KenSamson** [M17-Project/m17-tools#7] [Q&A] [answered]
Since I can no longer message over GitHub, one has to be creative on how one communicates here.

1)   Does this team have a way for me to donate a little?    I'm not rich, but I can buy you a coffee....
2)   Eventually ( I know everything takes a lot of time ) I want to be able to use Digirig with an FM VHF/UHF radio to talk on M17, and then also be able to connect on the internet to communicate with distant groups.   I have both Windows and Raspberry Pi / Linux machines.. and it seems like your work is striving to support both environments.

Thanks for all of your hard work.. it will be exciting  as this develops over time...

Ken Samson - KA0PIZ

**Comment by sp5wwp** [comment, marked as answer]
Hi!
1. Sure. We are a member project of the [Open Collective Foundation](https://opencollective.com/m17-project).
2. To use M17 with Digirig, you will need a proper 9600-baud radio. Unfortunately, not many handhelds support it. M17 is **not** based on AFSK.
